pkgs/games/itchiodl/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "itchiodl";
+  version = "2.1.2";
+
+  src = fetchFromGitHub {
+    owner = "Emersont1";
+    repo = "itchio";
+    rev = "v${version}";
+    hash = "sha256-U9oLrocwVdTqTQ26/MMiYO++yFGdOjnn8g+ea1jCl/A=";
+  };
+
+  format = "pyproject";
+
+  nativeBuildInputs = [
+    python3Packages.poetry
```

Comment:
`error: poetry was promoted to a top-level attribute, use poetry-core to build Python packages`